src/timeline/RoomlistModel.h

```diff
@@ -170,6 +170,10 @@ public slots:
     RoomPreview currentRoomPreview() const { return roomlistmodel->currentRoomPreview(); }
     void setCurrentRoom(QString roomid) { roomlistmodel->setCurrentRoom(std::move(roomid)); }
     void resetCurrentRoom() { roomlistmodel->resetCurrentRoom(); }
+    TimelineModel *getRoomById(const QString &id) const
+    {
+        return roomlistmodel->getRoomById(id).data();
+    }
```

Comment:
You don't call this anywhere?

Reply:
I misread that. You actually call this, I thought I was in the timelineviewmanager... Sorry!

Reply:
And I removed it and it built. Yikes!